CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
         * `get_pattern_pin_names` - [#1292](https://github.com/ni/nimi-python/issues/1292)
     * #### Changed
+        * `get_site_pass_fail` returns `dict` of `site`: `pass_fail` - [#1297](https://github.com/ni/nimi-python/issues/1297)
```

Comment:
Try to word this similar to the documentation, where it's really clear.
Suggestion:

`get_site_pass_fail` returns dictionary where each key is the site number and the value is a bool indicating pass / fail - [#1297](https://github.com/ni/nimi-python/issues/1297)